app/full_text_search.py

```diff
@@ -204,15 +264,9 @@ def add_record_to_index(person):
     """
     person_location_index = appengine_search.Index(
         name=PERSON_LOCATION_FULL_TEXT_INDEX_NAME)
-    name_params = [person.given_name,
-                   person.family_name,
-                   person.full_name,
-                   person.alternate_names]
-    names =  ':'.join([name for name in name_params if name])
     person_location_index.put(create_document(
```

Comment:
How about passing `person` object to create_document instead of the set of fields? Anyway you select the fields in create_document, right?